### DISCUSSION TITLE
How to make lookup table account for a versioned transaction?

### DISCUSSION THREAD

**Discussion by eddyson-B** [community/community#144026] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I have to make a versioned transaction that can be used in bundler, does anyone know how to do it?

**Comment by Rabnail-SOL** [comment, marked as answer]
You need to refer to this material
https://github.com/Rabnail-SOL/Solana-PumpFun-Bundler

**Reply by eddyson-B** [reply]
But there is one more thing I am unsure, when I create lookup table, I can't use it just at that time, why is it?

**Comment by Rabnail-SOL** [comment]
This is because there is cooldown time with Lookup table account, you need to wait about 10 seconds to use lookup table account after its creation.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Leo-SOL!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by Rabnail-SOL** [reply]
Thx
